lib/curl_threads.c

```diff
@@ -59,22 +59,30 @@ static void *curl_thread_create_thunk(void *arg)
   return 0;
 }
 
+#ifndef CURL_THREAD_STACK_SIZE
+#define CURL_THREAD_STACK_SIZE (100*1024)
```

Comment:
How did you come up with this number 100k? Seems low. Anyway just because that much isn't in use in libcurl code (as best we can ascertain) doesn't mean some called function absolutely won't consume it. It's virtual anyway I'd leave it alone. That reporter has a problem that it looks like is specific to him and he can adjust default stack size on his own.

Reply:
It's completely arbitrary, as I expect the default stack size to be as well. Why do you think it is low? The name resolver thread doesn't do much, why would it need a lot of stack?

Reply:
But yeah, figuring out what's sensible might turn out to be impossible to the extent that leaving it to its default makes more sense.

Clearly 100K also generates crashes in some situations...

Reply:
> The name resolver thread doesn't do much, why would it need a lot of stack?

It sounds like you're thinking only of libcurl and not the functions it calls in other libraries. We can't be expected to keep track of what other libraries may consume with their functions. It's virtual so I don't understand why mess with it. Is the reporter running out of virtual memory? It's not our problem.